components/PostCard.tsx

```diff
@@ -24,8 +24,6 @@ export default function PostCard({ post, isLinkActive }: PostCardProps) {
       background="secondaryBackground"
       overflow="hidden"
       borderRadius="lg"
-      mb="3"
-      mx="3"
```

Comment:
@Matthew-dw - Let me know if you are okay with me removing the extra spacing.

I don't think it's necessary for posts being displayed on the book page, and it helps with adding the posts with delete/edit buttons in the account page.

Reply:
Yes, its actually already been removed in my PR so its no problem